static/src/stylesheets/module/commercial/_adverts.scss

```diff
@@ -348,6 +348,10 @@
     &[open] .inline-plus {
         display: none;
     }
+
+    &:not([open]) > *:not(summary) {
```

Comment:
I can't get used to this. For me the complexity of this selector is too much. Look at it as if statement in the JS. if adverts__more && !open && (child !== summary) you see the point.